src/stdlib_experimental_system.F90

```diff
@@ -0,0 +1,37 @@
+module stdlib_experimental_system
+use, intrinsic :: iso_c_binding, only : c_int
+implicit none
+private
+
+interface
+#ifdef _WIN32
+subroutine system_sleep(ms) bind (C, name='Sleep')
+import c_int
+integer(c_int), value, intent(in) :: ms
+end subroutine system_sleep
+#else
+subroutine system_sleep(us) bind (C, name='usleep')
+import c_int
+integer(c_int), value, intent(in) :: us
+end subroutine system_sleep
+#endif
+end interface
+
+public :: sleep_millisec
```

Comment:
I would move this up in the module.

Is the `sleep_millisec` a good name? Perhaps `sleep_ms` is better? Or should this be just called `sleep`? How does C++ and Python handle this and what name do they use?

Reply:
The issue is generating compiler warnings because of their own `sleep()` implementations. I haven't looked into if they can be disabled. This may be an issue other such functions in this module would encounter.